### ISSUE TITLE
NVDA can't read the expanded content of a Primereact accordion with the Arrow-Down key

### ISSUE THREAD

**Issue by sylven32** [primefaces/primereact#1406]
**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request
```
Steps:
1. install NVDA, start it and open the NVDA Speech Viewer
2. browse to https://www.primefaces.org/primereact/showcase/#/accordion
3. expand Godfather1 (under the Default title, same happens with Multiple)
4. press Arrow-Down, once or several times
Expected:
NVDA should be able to read the summary of the movie
Actual:
Nothing happens, except that the page is scrolling down (standard feature of arrow key)

The noticeable workaround is to Shift-Tab after step 3 then Arrow-Down several times up to the movie's summary.

It's unclear whether NVDA is failing to read correct markup or if Primereact Accordion is setting incorrect aria attributes. Another defect open against NVDA (https://github.com/nvaccess/nvda/issues/11250) for this project's team to analyze.

Windows 10, Firefox 76.0.1

**Comment by sylven32** [comment]
NVDA team proposed a workaround with their tool. Hence closing this issue as primereact markup wasn't missing anything here.
